src/js/map/choropleth/choropleth.js

```diff
@@ -16,6 +21,18 @@ export class Choropleth extends Component {
         this.options = options;
         this.buffer = buffer;
         this.currentLayers = [];
+        this._choroplethRangeType = choroplethRangeTypes.BY_SUBINDICATOR;
```

Comment:
Ah so this sets the default range type

Reply:
yes, `BY_SUBINDICATOR` is the way it already works at the moment. so in default choropleth values will be calculated in the same way